ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/types.ts

```diff
@@ -0,0 +1,13 @@
+export type Severity = 'Critical' | 'Important' | 'Moderate' | 'Low';
```

Comment:
What do you think about using the `VulnerabilitySeverity` type from `types/cve.proto.ts` as the value here instead? It seems like that is the canonical type for CVE severities on the front end, and the values match what is expected in the search API according to the design doc.

There also looks to be value to display mappings in `messages/common.ts`.